docs/user_guide.md

```diff
@@ -86,6 +87,12 @@ _CodeChecker_. It analyzes your project and stores the reported code defects
 in a database, which can be viewed in a Web Browser later on (via `CodeChecker
 server`).
 
+With the server command start the server where the results will be stored and
```

Comment:
I would somehow merge this with the previous paragraph. `CodeChecker server` is mentioned twice.